### ISSUE TITLE
Create issues for The Android Wheel (tm)

### ISSUE THREAD

**Issue by amulware** [toggl/mobileapp#4030]
Essentially a copy of the iOS time editing view.

Notable differences:

- instead of iOS barrels, we'll use native Android pickers; since we've had a lot of troubles adapting these to our own design in the past, we probably want them in popups, not inside our own UI

- actually, I think that's the main thing

Before we can start on this we need:

- someone to split this into multiple issues if possible (and in either case, create all issues necessary, and add them to the Giskard 1.11 project)
- designs
- rxfactoring the VM (scheduled in Giskard 1.10)

**Comment by amulware** [comment]
Have all issues been created? @daividssilverio @kornelijepetak?

**Comment by kornelijepetak** [comment]
One is missing and I'm working on it right now.

**Comment by kornelijepetak** [comment]
As the matter of fact, the issue I wanted to create pertains to tracking, and after a little bit of research into our codebase, we seem to cover what we need already. That is implemented in Foundation, so this is actually not required.

From the outstanding issues, I think we're covered.
